src/components/shared/Notifications.tsx

```diff
@@ -102,11 +103,19 @@ export const NotificationComponent = ({
 
 	return (
 		<div className={cn(notification.type, "alert sticky")}>
+			{notification.type === "warning" &&
+				<LuTriangleAlert className="warning-symbol-warning" />
+			}
+			{notification.type === "error" &&
+				<LuTriangleAlert className="warning-symbol-error" />
+			}
 			{closeNotification &&
 				<ButtonLikeAnchor
 					onClick={() => closeNotification(notification.id)}
-					className="fa fa-times close"
-				/>
+					className="close"
+				>
+					<LuX />
+				</ButtonLikeAnchor>
 			}
 			<p>{t(notification.message, notification.parameter)}</p>
 		</div>
```

Comment:
This is quite off-center. I'd put the warning triangle inside the `<p>` and give the icon a size of ~17px.
```suggestion
        <div className={cn(notification.type, "alert sticky")}>
			{closeNotification &&
				<ButtonLikeAnchor
					onClick={() => closeNotification(notification.id)}
					className="close"
				>
					<LuX />
				</ButtonLikeAnchor>
			}
			<p>
			    {(notification.type === "warning" || notification.type === "error") &&
				    <LuTriangleAlert className="warning-symbol-warning" style={{ fontSize: 17 }} />
			    }
			    {t(notification.message, notification.parameter)}
			</p>
		</div>
```

Reply:
Could you point me to which notifications exactly you see the issue with?

Reply:
Yeah, sorry, should have included a screenshot. Took me a while to find this again myself^^
<img width="1589" height="550" alt="Bildschirmfoto 2025-08-14 um 11 09 21" src="https://github.com/user-attachments/assets/93ef09b9-a15b-4fee-abdc-f6862150a224" />

For single line warnings this isn't as bad, but this fix should work for those as well.